pcdsdevices/lasers/rfof.py

```diff
@@ -13,17 +13,242 @@
 
 
 class CycleRfofRx(BaseInterface, Device):
-    """
-    Class for reading back the Cycle RFoF receiver PVs.
-    """
+
+    """Class for reading back the Cycle RFoF receiver PVs."""
+
     rf_temperature = Cpt(EpicsSignalRO, ':RFTEMPERATURE', kind='normal')
     rf_power = Cpt(EpicsSignalRO, ':RFPOWER', kind='normal')
     optical_power = Cpt(EpicsSignalRO, ':OPTICALPOWER', kind='normal')
 
 
 class CycleRfofTx(CycleRfofRx):
-    """
-    Class for reading back the Cycle RFoF transmitter PVs.
-    """
+
+    """Class for reading back the Cycle RFoF transmitter PVs."""
+
     # Transmitter adds a single control PV
     enable_laser = Cpt(EpicsSignal, ':ENABLELASER', kind='normal')
+
+
+"""Instrumentation Technologies Low-Jitter Continuous Wave"""
```

Comment:
These lines don't do anything - where do they belong?

Reply:
The blank lines? Or the Cycle RFOF lines?

Reply:
Ah, yes. I thought the triple quotes also acts as a comment

Reply:
Sometimes people do put strings in the source code instead of comments, particularly with long multiline comments. I think some Python tutorials mention this. There is a small difference in loading and it's less clear the intent but there isn't much of a functional difference otherwise.